backend/game_engine/common.py

```diff
@@ -187,7 +194,18 @@ def _to_dict_item(self, v):
                 'Serializable object contains unsupported attribute {}'.format(v))
 
     def to_dict(self):
-        log_message('to_dict called for {}'.format(self.__class__))
+        def massage_for_logging(attrs_dict):
```

Comment:
I think the call to log_message within the to_dict method will be too expensive and we can likely remove the call entirely here to simplify this. I initially had it here for debugging during the initial development.